cumulusci/robotframework/CumulusCI.py

```diff
@@ -300,15 +321,19 @@ def set_test_metric(self, metric: str, value=None):
         The keyword takes a metric name, which can be any string, and a value, which
         can be any number.
 
-        Using this keyword will automatically add the tag cci_metric_metric to the test case
+        Using this keyword will automatically add the tag cci_metric to the test case
```

Comment:
I don't see what value this has. For example, if one test has the metric "foo" and one has the metric "bar", those two will be rolled up in the statistics table as if they were the same thing. Is that the intent, or should this tag be added to `tagstatinclude`?

You might want to add a sentence or two explaining what this tag is used for.

Reply:
Added to tagstatinclude.

And added this sentence: "These permit downstream processing in tools like CCI and MetaCI."